inception-project/inception
# #848 - Support Jena/Fuseki FTS mode

**What's in the PR**
- Added support for the Fuseki FTS implementation
- Updated ZBW SPARQL profiles to use Fuseki FTS
- Added tests running SPARQLQueryBuilder against ZBW endpoints using FTS
- Added class to help building RDF collections compatible with the SparqlBuilder from RDF4J

**How to test manually**
* Create a KB based on one of the ZBW profiles
* Check that FTS is enabled after KB creation
* Try linking entities and/or searching for KB items on the KB page

**Automatic testing**

**Documentation**
